modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,23 @@
+= B2B Solutions
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: b2b, edi, portal
+
+== B2B (Business-to-Business)
+
+B2B in MuleSoft is the use of API's or Electronic Data Interchange (EDI) by enterprises to carry out business transactions such as purchasing goods from each other. B2B messages are typically sent over transports such as SFTP and exchanges such as AS2. MuleSoft's B2B solutions integrate seamlessly into the Mule runtime.
+
+== About this Page
+
+This page introduces MuleSoft's B2B Solutions and links to pages that provide details about Anypoint Partner Manager and MuleSoft's EDI message packs. In addition, we provide configuration details for implementing B2B Solutions in Anypoint Studio's Graphical Editor.
```

Comment:
Unless B2B Solutions is a feature, Solutions shouldn't be capitalized except in a title. What is a b2b solution that's not in Anypoint Partner Manager? What's a message pack (how does it related to message flow, etc.)

Reply:
Please stop referencing the doc in the doc.

Reply:
This has changed per the PM>